### PR TITLE
Windows kernelspace!!!

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

File: backends/windows/kernelspace/sink.go [hunk 1]
```diff
@@ -104,7 +104,9 @@ func (s *Backend) DeleteEndpoint(namespace, serviceName, key string) {
 }
 func (s *Backend) SetService(svc *localnetv1.Service) {}
```
Comment (jayunit100) [inline backends/windows/kernelspace/sink.go:105]: i think we'll need to impl `SetService` and/or `SetEndpoint` , at the very minimum, before doing any meaningful testing

File: backends/windows/kernelspace/proxier.go [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+		// TODO: Add Back the flag
+		//if len(hostMac) == 0 {
+		//	return nil, fmt.Errorf("could not find host mac address for %s", nodeIP)
+		//}
+	}
+
+	isIPv6 := netutils.IsIPv6(nodeIP)
+	myProxier := &Proxier{
+		endPointsRefCount:   make(endPointsReferenceCountMap),
+		serviceMap:          make(ServicesSnapshot),
```
Comment (jayunit100) [inline backends/windows/kernelspace/proxier.go:203]: dimitri added these to align it w/ the upstream kernelspace stuff

**Comment by jayunit100** [comment]
@papagalu i can rebase this - but are you ok w it as it stands to merge ?

**Comment by jayunit100** [comment]
@papagalu looks like rebased is all set 

can you leave links to follow ons / thoughts on how you did the testing here ? thank you for helping us finish this!!!!!!!!!!!!

**Comment by papagalu** [comment]
I believe we can merge this as it is right now, we still have #322 and #323 to fix

File: backends/windows/kernelspace/proxier_events.go [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+	discovery "k8s.io/api/discovery/v1"
+	netutils "k8s.io/utils/net"
+
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/kpng/api/localnetv1"
+)
+
+// OnEndpointsAdd is called whenever creation of new windowsEndpoint object
+// is observed.
+//func (proxier *Proxier) OnEndpointsAdd(ep *localnetv1.Endpoint, svc *localnetv1.Service) {
```
Comment (knabben) [inline backends/windows/kernelspace/proxier_events.go:34]: delete these lines?

File: backends/windows/kernelspace/OWNERS [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```
Comment (knabben) [inline backends/windows/kernelspace/OWNERS:3]: add @papagalu

**Pull request merged**